content/_changelogs/12.0.0.md

```diff
@@ -1,20 +1,191 @@
 ## 12.0.0
 
-_Released MM/DD/YYYY_
+_Released 12/06/2022_
+
+**Summary:**
+
+The Session and Origin Experiment is now generally available for end-to-end
+testing. This means Cypress now fully supports:
+
+- testing multiple domains of different origins in a single test with the new
+  [`cy.origin()`](/api/commands/origin)
+- caching and restoring `cookies`,
+  [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage),
+  and
+  [`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
+  between tests
+- configuring
+  [`testIsolation`](/guides/core-concepts/writing-and-organizing-tests#Test-Isolation)
+  in suites to define whether or not the browser context is cleaned or persisted
+  between tests
+
+Additionally in this release, enhancements were made to how Cypress manages DOM
+element resolution to reduce the likelihood of hitting the dreaded detached DOM
+errors due to maintaining stale DOM references. We've updated our
+[Retry-ability Guide](/guides/core-concepts/retry-ability) with all the details
+if you'd like to learn more.
+
+Read more about 12.0 in
+[our blog post](https://cypress.io/blog/2022/12/06/cypress-12-release/).
 
 **Breaking Changes:**
 
-- `experimentalSessionAndOrigin` flag has been removed and all functionality is
-  on by default now.
-- The new concept of `testIsolation` has been introduced and it defaults to
-  `on`.
-- Cypress now throws an error if commands are invoked from inside a `.should()`
-  callback. This previously resulted in unusual and undefined behavior; it is
-  now explicitly an error.
-- `Cookies.defaults` and `Cookies.preserveOnce` have been removed. Please update
-  to use [`cy.session()`](/api/commands/session) to preserve session details
+- Cypress dropped support for Node.js 12, 15 and 17 because they these versions
+  have reached end-of-life. Installing Cypress on your system now requires
+  Node.js 14, 16 or 18+. Addressed in
+  [#24885](https://github.com/cypress-io/cypress/pull/24885).
+- Cypress has always pushed for writing tests in a clean slate. In Cypress 12,
+  we enforce running tests in a clean browser context through the
+  [test isolation](/guides/core-concepts/writing-and-organizing-tests#Test-Isolation).
+  This option is configurable, but is enabled by default behavior. Whether
```

Comment:
"Whether enabled or disabled, this changes how Cypress previously cleaned up the browser context **before** each test and you may experience test errors with this upgrade."